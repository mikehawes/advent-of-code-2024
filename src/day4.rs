mod find_cursor;
mod lines;
pub mod word_search;
