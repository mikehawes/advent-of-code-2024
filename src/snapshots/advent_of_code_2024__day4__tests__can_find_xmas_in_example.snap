---
source: src/day4.rs
expression: "print_relevant_points(&word_search, &points)"
snapshot_kind: text
---
....XXMAS.
.SAMXMS...
...S..A...
..A.A.MS.X
XMASAMX.MM
X.....XA.A
S.S.S.SXSS
.A.A.A.A.A
..M.M.M.MM
.X.X.XMASX
